### PR TITLE
makefile for style

### PR DESCRIPTION
**Pull request by debajyotidatta**
The instructions [here ](https://github.com/bigscience-workshop/biomedical/blob/master/CONTRIBUTING.md) require you to run the following:

`make style
make quality`

But that can't be run without a Makefile.

So this PR just adds the Makefile.

### CODE REVIEW

**Review by galtay — APPROVED** [review]

**Review by galtay — CHANGES_REQUESTED** [review]
do we want to change `check_dirs := promptsource` ?

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# https://github.com/bigscience-workshop/promptsource/blob/main/Makefile
+
+.PHONY: quality style
+
+check_dirs := promptsource
+
+# Check that source code meets quality standards
+
+quality:
+	black --check --line-length 119 --target-version py38 $(check_dirs)
```
Comment (galtay) [inline Makefile:11]: black recently announced a more stable versioning schedule https://black.readthedocs.io/en/stable/change_log.html  ... should we update the pinned version in dev requirements to `22.1.0` ?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# makefile from the huggingface promptsource repository
+# https://github.com/bigscience-workshop/promptsource/blob/main/Makefile
+
+.PHONY: quality style
+
+check_dirs := promptsource
```
[the remark this answers is not in the document]
Comment (debajyotidatta) [reply Makefile]: Yes! Good catch. I added the two code folders. We probably should put all code in a folder inside a main folder.

File: Makefile (hunk 1 above)
Comment (debajyotidatta) [reply Makefile:11]: Yes, definitely! I have upgraded it. @hakunanatasha : Are you okay with this change?

**Review by galtay — APPROVED** [review]

**Comment by galtay** [comment]
looks like there are some merge conflicts to resolve but thanks for updating check_dirs and the black version. might be easier to make this PR just the Makefile and requirements file and handle the actual formatting changes of the `*.py` files in another PR but I'm fine either way.

**Review by hakunanatasha — CHANGES_REQUESTED** [review]
@debajyotidatta i'm going to use the make file but we need to set some rules for black; if you don't use the flag --skip-string-normalization will automatically convert single quotes to double quotes (which is what you're seeing in all these files).

While this is not destructive for the remaining files, it will be destructive in the future hence I'm reverting my approval (moreover there have been changes on those files; there will be a merge conflict mess).

File: Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (hakunanatasha) [reply Makefile]: yep! will do

**Review by hakunanatasha — APPROVED** [review]
I'm going to merge and make edits in a new branch

**Pull request merged**